### ISSUE TITLE
[semantic-issue] Add semantics for sklearn GridSearchCV

### ISSUE THREAD

**Issue by osryde** [spangea/Pyra#177]
**Description**
Invokes the constructor to instantiate a GridSearchCV object.
Notebook: 0-91-accuracy-score-for-4-models-lr-rc-gbc-rfc.py

**Minimal running example**
```
from sklearn.model_selection import GridSearchCV

grid_search = GridSearchCV(estimator=model, param_grid=grid, n_jobs=-1, cv=cv, scoring='accuracy', error_score=0)
```

**Links to documentation**
https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.GridSearchCV.html

**Note (optional)**
Returns an object of GridSearchCV type.
